### PR TITLE
Make the code more concise

### PR DESCRIPTION
**Pull request by Lete114**
Don't write long judgments in if 
Reference: [Use descriptive conditions](https://github.com/felixge/node-style-guide#use-descriptive-conditions)

### CODE REVIEW

File: lib/defaults.js [hunk 1]
```diff
@@ -16,13 +16,14 @@ function setContentTypeIfUnset(headers, value) {
...
-    // For node use HTTP adapter
-    adapter = require('./adapters/http');
-  }
+  // For browsers use XHR adapter
+  var xhr = typeof XMLHttpRequest !== 'undefined';
+  // For node use HTTP adapter
+  var http = typeof process !== 'undefined' && Object.prototype.toString.call(process) === '[object process]';
+
+  if (xhr) adapter = require('./adapters/xhr');
+  if (http) adapter = require('./adapters/http');
```
Comment (suyundukov) [inline lib/defaults.js:25]: Just wondering, you sure we don't need an `else if` here?

```js
if (xhr) {
  adapter = require('./adapters/xhr');
} else if (http) {
  adapter = require('./adapters/http');
}
```

Comment (lete114) [reply lib/defaults.js:25]: I'm sure , We don't need an `else if `
There will be no XMLHttpRequest in Node and no HTTP in the browser

Comment (lete114) [reply lib/defaults.js:25]: Do you have a way for Node to get browsers that have both XMLHttpRequest and HTTP ? 
I don't think that's going to happen

**Review by caugner — CHANGES_REQUESTED** [review]

**Comment by jasonsaayman** [comment]
Hi this seems to be more of a style / semantics change, with Axios being as old as it currently is and a new better upgrade on its' way I think this is not a necessary change for now. Thanks for the contribution and I encourage you to open a new pull request if you want on `next` when i eventually get to creating it (this year still)

**Pull request closed without merging**